libgnucash/engine/mocks/fake-qofquery.cpp

```diff
@@ -0,0 +1,155 @@
+#include <config.h>
+
+#include <list>
+
+#include "fake-qofquery.h"
+#include "gmock-qofbook.h"
+
+
+
+/* class QofFakeQueryPool */
+
+static class QofFakeQueryPool
+{
+public:
+    QofFakeQueryPool() {};
+
+    void addQuery(QofFakeQuery *query)
```

Comment:
Camel case is for type names only, use snake case for function names. See our [C++ Coding Guide](https://wiki.gnucash.org/wiki/CodingStandard#C.2B.2B_Format)

Reply:
Oh, this means I have to change the function names of all mock class methods now, right?

Reply:
Yes.

Reply:
Done